homeassistant/components/template/strings.json

```
@@ -244,13 +306,19 @@
           "options": "Available options"
         },
         "data_description": {
-          "device_id": "[%key:component::template::common::device_id_description%]"
+          "device_id": "[%key:component::template::common::device_id_description%]",
+          "state": "Template for the select’s current value.",
+          "select_option": "Defines actions to run to select an `option` from the `options` list. Receives variable `option`.",
```

"when" missing, too? Perhaps:

```suggestion
          "select_option": "Defines actions to run when an `option` from the `options` list is selected. Receives variable `option`.",
```